src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/LinkerTests.cs

```diff
@@ -394,5 +394,28 @@ public unsafe bool MyMethod (Android.OS.IBinder windowToken, [global::Android.Ru
 				}
 			}
 		}
+
+		[Test]
+		public void TypeRegistrationsFallback ([Values (true, false)] bool enabled)
+		{
+			if (!Builder.UseDotNet)
+				return;
```

Comment:
If you make other changes here, you might put `Assert.Ignore("Test only valid on .NET 6")` or something.

Reply:
Updated.